olive/passes/onnx/extract_adapters.py

```diff
@@ -37,7 +37,7 @@ class ExtractAdapters(Pass):
 
     @classmethod
     def _default_config(cls, accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
-        return {
+        config = {
```

Comment:
I did not see whether to run matmul 4 bits in this pass? The matmul 4 bits quantization is run before this pass?

Reply:
As discussed offline, this pass expects the in4 quantization pass to be run before it if the user wants an in4 quantized model.